samwise.js

```diff
@@ -5,49 +5,48 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.